### PR TITLE
Fixed Message Parsing and Missing GEDs

### PR DESCRIPTION
**Pull request by eironside**
Fixed issue with GES restart and missing GEDs (guids dont align, have to
lookup by name/owner or name). [Issue #1617](https://devtopia.esri.com/WebGIS/arcgis-geoevent/issues/1617)
Fixed issue [#2011](https://devtopia.esri.com/WebGIS/arcgis-geoevent/issues/2011) - Invalid LAT (off by 10) and #2 #
Fixed the issue reported above as well as many other parsing issues with
the messages:
TAIP0xF1(CP): fixed LAT conversion
TAIP0xF2(LN): fixed LAT conversion
TAIPRAM(AM): Fixed time, LAT, LON, speed, heading, alarm code, and
source conversions

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,10 +44,9 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>2.5.1</version>
 					<configuration>
-						<source>1.7</source>
-						<target>1.7</target>
+						<source>8</source>
+						<target>8</target>
```
Comment (hanoch) [inline pom.xml]: To support GES 10.4, we can build with JDK 7 (1.7) and run on JRE 7 or JRE 8.

**Review by hanoch — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (eironside) [reply pom.xml]: On my system, it seems to be the other way around??  i just tested deploying it to a 10.4 system and it worked fine as is (compiled with target 8).

Comment (hanoch) [reply pom.xml]: Is 10.4 already running with JRE 8?
Was there an issue when building with 1.7 and running with 10.4?
If we switch to JDK 8 we won't be able to be compatible with GES versions that are running with JRE 7 (10.2, 10.3?).

Comment (eironside) [reply pom.xml]: I've been compiling against JDK 1.8 for my Gallery Item releases with the goal of being compatible with as many versions as possible starting with 10.7 and going backwards. In my testing, 10.4 and above are compatible with compiling at JDK 1,8.  Versions 10.3 and before don't run and are not compatible.

Comment (hanoch) [reply pom.xml]: Is there any issue building with JDK 7/1.7 and running on JRE 8 (10.4 and above)?
If not, we should/can continue to build with JDK 7/1.7.

**Pull request merged**